cli/cli.go

```diff
@@ -27,13 +27,10 @@ type Interface interface {
 // cli implements cli.Interface
 type cli struct {
 	Configuration
-	version    string
-	build      string
```

Comment:
Unfortunately, I did a squash on that part of the history, but it's probably my fault. By adding the fields to Configuration to streamline `NewCLI`, they are now mutable. It's nothing earth-shattering, but I'd prefer that Configuration then extend the same constraint with a [getter](https://golang.org/doc/effective_go.html#Getters), delegated via cli. But since it wasn't you that broke the abstraction, I won't ask you to fix it (unless you want to). ;)